src/common/snippets/tests/include/lowered/pass/buffer_allocation.hpp

```diff
@@ -52,13 +54,6 @@ class EltwiseBufferAllocationTest : public BufferAllocationTest {
     std::shared_ptr<ov::Model> GetModel() const override;
 };
 
-class MHABufferAllocationTest : public BufferAllocationTest {
-protected:
-    std::shared_ptr<ov::Model> GetModel() const override;
-
-    static void MarkBrgemm(const std::shared_ptr<ov::snippets::op::Brgemm>& node, const std::vector<size_t>& subtensor);
-};
```

Comment:
Yeah, it's moved to cpu specific part because common Snippets pipeline doesn't support Brgemm blocking (this pass is registered in CPU backend side). Because of that `InitLoops` incorrectly calculates `work_amount` (second input of Brgemm has `is_incremented_true` in loop by M)